taichi/ir/statements.cpp

```diff
@@ -94,6 +91,19 @@ void OffloadedStmt::all_blocks_accept(IRVisitor *visitor) {
     tls_epilogue->accept(visitor);
 }
 
+bool is_clear_list_task(const OffloadedStmt *stmt) {
+  return (stmt->task_type == OffloadedStmt::TaskType::serial) &&
+         (stmt->body->size() == 1) && stmt->body->back()->is<ClearListStmt>();
+}
+
+ClearListStmt::ClearListStmt(SNode *snode) : snode(snode) {
+  TI_STMT_REG_FIELDS;
+}
+
+std::unique_ptr<Stmt> ClearListStmt::clone() const {
+  return Stmt::make_typed<ClearListStmt>(snode);
+}
```

Comment:
nit: I think this is unnecessary -- with `TI_STMT_DEF_FIELDS(ret_type, snode);`, `TI_DEFINE_ACCEPT_AND_CLONE` should be fine?

Reply:
Thanks! Done